### ISSUE TITLE
hiddenore changes

### ISSUE THREAD

**Issue by Convoy20** [DevotedMC/Rhea-Public#1]
https://github.com/DevotedMC/Rhea-Public/blob/main/templates/categories/nether/HiddenOre/config.yml.j2
GitHub
Rhea-Public/config.yml.j2 at main · DevotedMC/Rhea-Public
Contribute to DevotedMC/Rhea-Public development by creating an account on GitHub.

first off just remove most of that config RE: ores and copy the civclassic one as a starter
second I want you to expand the variables in this file:  - { name: 'alpha', categories: ['nether','starter'], port: 'x', nice_name: 'Asmodeus', limit_monster: '80', ticks_monster: '200', lagg_mob: '5000', bastion: '3.35', bricks: '40', iron: '200', gold: '400', diamond: '1500', netherite: '4000'}
https://github.com/DevotedMC/Rhea-Public/blob/main/variables/all.yml
GitHub
Rhea-Public/all.yml at main · DevotedMC/Rhea-Public
Contribute to DevotedMC/Rhea-Public development by creating an account on GitHub.

and make all of the mining notes be variables
and all the ore rates
basically you define it in that section I mentioned, then in the config you do something like hit_points: {{ shard.iron }}
